server/i18n/en/errors.json

```diff
@@ -93,23 +93,13 @@
   "startDate": {
     "empty": "You must enter a start date",
     "invalid": "The start date is an invalid date",
-    "conflict": "This bedspace is not available for these dates"
-  },
-  "outOfServiceFrom": {
-    "empty": "You must enter a start date",
-    "invalid": "The start date is an invalid date",
-    "conflict": "This bedspace is not available for these dates"
+    "conflict": "This booking is not available for these dates"
   },
   "endDate": {
     "empty": "You must enter a end date",
     "invalid": "The end date is an invalid date",
     "beforeStartDate": "The end date must be before the start date",
-    "conflict": "This bedspace is not available for these dates"
-  },
-  "outOfServiceTo": {
-    "empty": "You must enter an end date",
-    "invalid": "The end date is an invalid date",
-    "conflict": "This bedspace is not available for these dates"
+    "conflict": "This booking is not available for these dates"
```

Comment:
I made these changes on a seperate branch and ran into the same realisation as detailed in your commit message. 
The simplest suggestion I can think of is namespacing the error messages so rather than  
``` { ... 
 "endDate": {
   "conflict": "... error message..."
}
```   
we have: 
``` { ... 
"outOfServiceBeds": {
 "endDate": {
   "conflict": "... error message..."
  }
"bookings":{
 "endDate": {
   "conflict": "... error message..."
  } 
 }
```
But I imagine this will require quite a lot of work in the API to implement.  
Specifically for the `conflict` error we could lean on [this component](https://github.com/ministryofjustice/hmpps-approved-premises-ui/pull/780/commits/718e5bd292598c6dae76f4da9ba5a3668324b860) but the approach doesn't scale.

Reply:
yes I was thinking that the API could use the model name when generating this error messages. But I'm not sure how we'd introduce that change smoothly across all 3 CAS services...

I'll bring this up in our next x-CAS API meeting...